lib/compute/index.js

```diff
@@ -312,6 +318,96 @@ Compute.prototype.createNetwork = function(name, config, callback) {
 };
 
 /**
+ * Create a global forwarding rule.
+ *
+ * @resource [GlobalForwardingRule Resource]{@link https://cloud.google.com/compute/docs/reference/v1/globalForwardingRules#resource}
+ * @resource [GlobalForwardingRules: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/globalForwardingRules/insert}
+ *
+ * @param {string} name - Name of the rule.
+ * @param {object} config - See a
```

Comment:
Yar.. since we are redirecting the user to upstream docs, when I see *a*, it almost makes me assume there are several upstream docs called the same thing (which doesn't really make sense). Opposed to *the* which would kind of imply it's the definitive resource for GlobalForwardingRules.

Mostly a nitpick, so feel free to ignore if you disagree!